README.md

```diff
@@ -1,13 +1,13 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project is a full-stack authentication project with a simple registration and log in flow. The user is authenticated using tokens and passwords are stored securily in a dababase. Forms to register are built in the front end part. Only when the user is logged in they are able to view the protected content.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The styling is unfortunately not yet finished. I run into problems with my styled components and the absolute and relative positionings which I finally solved with z-index. The styling will be developed more, same with returning something meaningful from the api.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+FE: https://tender-cori-4354e6.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```


![image](https://user-images.githubusercontent.com/50926159/120919812-cd457f80-c6bb-11eb-883a-1abc973179d3.png)